configs/mars_example_config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "client": "mars",
+    "dataset": "ecmwf-mars-output",
+    "target_filename": ".nc",
+    "target_path": "gs://ecmwf-downloads/hres-single-level",
+    "append_date_dirs": "true",
+    "partition_keys": "date"
+  },
+
+  "selection": {
+    "stream": "oper",
+    "levtype": "sfc",
+    "param": "10fg6/10u/10v/100u/100vcrr/2t/2d/200u/200v/cp/dsrp/hcc/i10fg/lcc/lsp/lspf/lsrr/msl/ptype/sf/sp/ssr/tcrw/tclw/tcsw/tcw/tcwv/tp",
```

Comment:
Please use a standard JSON array instead of the `/` delimiter syntax. Same with the `step` below.

Reply:
hey, @alxmrs, done with the desired changes replaced all delimiters with the array.